### PR TITLE
feat(slider): Implement daisyUI example

### LINKED ISSUE #620: Add slider daisyUI examples
**Description**
Provide usage of slider headless to display a daisyUI slider.

### PR DESCRIPTION
**Pull request by andreascorti**
closes #620

### CODE REVIEW

File: angular/demo/daisyui/src/app/samples/slider/slider.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<input
+	[auUse]="widget.directives.sliderDirective"
+	type="range"
+	[min]="state().min"
+	[max]="state().max"
+	[value]="state().sortedHandles[0].value"
+	[step]="state().stepSize"
+	[class]="'range'"
```
Comment (quentinderoubaix) [inline angular/demo/daisyui/src/app/samples/slider/slider.component.html:8]: Maybe
```angular
<input class="range" [class]="className"/>
```
Like this a user in stackblitz could use the slider like this:
```angular
   <app-slider min="0" max="100" stepSize="1" [values]="values" (valuesChange)="valuesChange($event)" className="range-success" />
```
and have a different color

File: demo/src/routes/docs/[framework]/daisyUI/slider/+layout.server.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import {daisyUIMetadata} from '$lib/components-metadata';
+
+export const load = async () => {
+	return {
+		...daisyUIMetadata.Slider,
+		prev: {
```
Comment (quentinderoubaix) [inline demo/src/routes/docs/[framework]/daisyUI/slider/+layout.server.ts:6]: You also need to update the prev tag of +layout.server.ts file of Accordion to target the slider.

File: demo/src/routes/docs/[framework]/daisyUI/slider/headless/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+	import daisyuiSample from '@agnos-ui/samples/daisyui/slider/default';
+	import Sample from '$lib/layout/Sample.svelte';
+	import Section from '$lib/layout/Section.svelte';
+</script>
+
+<Section label="DaisyUI" id="default" level={2}>
+	<Sample title="DaisyUI example" sample={daisyuiSample} height={395} />
```
Comment (quentinderoubaix) [inline demo/src/routes/docs/[framework]/daisyUI/slider/headless/+page.svelte:8]: height should be 114

File: react/demo/src/daisyui/samples/slider/Default.route.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import {useState} from 'react';
+import {Slider} from './Slider';
+
+const TailwindDemo = () => {
```
Comment (quentinderoubaix) [inline react/demo/src/daisyui/samples/slider/Default.route.tsx:4]: Rename to DaisyUIDemo or smth more meaningful, TailwindDemo is not a great name.

File: react/demo/src/daisyui/samples/slider/Slider.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+			type="range"
+			ref={sliderSetRef}
+			min={state.min}
+			max={state.max}
+			value={state.sortedHandles[0].value}
+			onChange={(e) => null}
+			step={state.stepSize}
+			className="range"
+			aria-label={state.sortedHandles[0].ariaLabel}
+			onClick={(e) => click(e as unknown as MouseEvent)}
```
Comment (quentinderoubaix) [inline react/demo/src/daisyui/samples/slider/Slider.tsx:26]: You could not use `onClick={(e) => click(e.nativeEvent)}` here ?

File: svelte/demo/src/daisyui/samples/slider/Slider.svelte [hunk 6]
```diff
@@ -0,0 +1,44 @@
+<script lang="ts" context="module">
+	import type {SliderProps as Props} from '@agnos-ui/svelte-headless/components/slider';
```
Comment (quentinderoubaix) [inline svelte/demo/src/daisyui/samples/slider/Slider.svelte:2]: No need to put the imports in a module context.

**Review by quentinderoubaix — COMMENTED** [review]
Some minor comments, thanks for the PR @andreascorti !

File: angular/demo/daisyui/src/app/samples/slider/slider.component.html (hunk 1 above)
Comment (andreascorti) [reply angular/demo/daisyui/src/app/samples/slider/slider.component.html:8]: It is already the case, `className` is applied by `sliderDirective`

**Review by quentinderoubaix — APPROVED** [review]
LGTM !

**Pull request merged**